addons/web_editor/i18n/web_editor.pot

```diff
@@ -4,10 +4,10 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 16.0beta\n"
+"Project-Id-Version: Odoo Server 16.0\n"
```

Comment:
this file contains no other changes than the comments

Reply:
My comment was to minimise the diff. I expect it will be gone in the next update with some actual content, it's only comments anyway.